pipeline.cpp

```diff
@@ -79,7 +96,39 @@ void DrawLine(Buffer2D<PIXEL> & target, Vertex* const triangle, Attributes* cons
  ************************************************************/
 void DrawTriangle(Buffer2D<PIXEL> & target, Vertex* const triangle, Attributes* const attrs, Attributes* const uniforms, FragmentShader* const frag)
 {
-    // Your code goes here
+    // Bounding Box
+    int top = std::max(triangle[0].y, std::max(triangle[1].y, triangle[2].y));
+    int bottom = std::min(triangle[0].y, std::min(triangle[1].y, triangle[2].y));
+    int right = std::max(triangle[0].x, std::max(triangle[1].x, triangle[2].x));
+    int left = std::min(triangle[0].x, std::min(triangle[1].x, triangle[2].x));
```

Comment:
In definitions.h Brother Christensen gave some functions that we could use for these min and max calls. They do the same thing, but there may be a future assignment that we can use another one that he gives us that isn't available in the standard library. Just FYI.